distribution/bin/enso

```diff
@@ -8,5 +8,5 @@ for opt in "$@"; do
 done
 
 
-exec java --module-path $COMP_PATH $EXTRA_OPTS $JAVA_OPTS -m org.enso.runtime/org.enso.EngineRunnerBootLoader "$@"
+exec java --module-path $COMP_PATH --add-modules org.jline $EXTRA_OPTS $JAVA_OPTS -m org.enso.runtime/org.enso.EngineRunnerBootLoader "$@"
```

Comment:
`jline-3.23.0.jar` is an *automatic module*. We have to explicitly add it to the `--add-modules` option, otherwise the JVM will not see it in the boot module layer.

Reply:
I think this change should also be mirrored in
https://github.com/enso-org/enso/blob/develop/lib/scala/runtime-version-manager/src/main/scala/org/enso/runtimeversionmanager/runner/Runner.scala#L189-L194

Reply:
Well, it is a good idea, not necessary since we don't use jline in language-server, neither in runtime, but at least to be consistent.  Added in 7d1feedebc1b98c4d30d43a51c194b4bf5779053